lib/reporters/base.js

```diff
@@ -317,10 +322,14 @@ function Base (runner) {
  * the bundled reporters.
  *
  * @api public
+ *
+ * @param {Object} options
+ * @param {[boolean]} options.listFailures - if false, will not output failure information
```

Comment:
I haven't seen this syntax before, but I believe the following works (and it also shows what the default is; also `options` is optional):

```js
/*
 * @param {Object} [options] - Options
 * @param {boolean} [options.listFailures=true] - If false, will not output failure information
 */
```